conf/ocsci/aws_ipi_1az.yaml

```diff
@@ -0,0 +1,7 @@
+---
```

Comment:
Also please put some comment at the start of the config file like:
`# This config is for setup cluster in 1 Availability Zone for AWS`

Thanks

Reply:
Done